src/index.ts

```diff
@@ -1,6 +1,19 @@
-export const sum = (a: number, b: number) => {
-  if ('development' === process.env.NODE_ENV) {
-    console.log('boop');
+import { JolocomPlugin, JolocomSDK } from '@jolocom/sdk'
+import { PREFIX } from './constant'
+import { StacksDidMethod } from './didMethod'
+import { StacksNetwork } from '@stacks/network'
+
+export class JolocomStacksDidMethodPlugin implements JolocomPlugin {
+  private readonly stacksNetwork?: StacksNetwork
+
+  constructor(stacksNetwork?: StacksNetwork) {
```

Comment:
can we also pass the `prefix` as a constructor argument (with default as `PREFIX`) and then pass that through to the constructors of `StacksDidMethod` below (which should also pass it through to `StacksResolver` and `StacksRegistrar`) ?

This is so that it can be easily changed if someone wants to mess with it